docs/exploration-and-organization/exploration.md

```diff
@@ -6,77 +6,63 @@ redirect_from:
 
 # Basic exploration
 
-## See what your teammates have made
+How to poke around your Metabase.
```

Comment:
it would be nice to a small introduction what is the expectation of reading this page, it's full of really important information for users to move around metabase.

Reply:
Actually this will be a good topic for a video :D